etn-ccis/blui-react-workflows
# Feature/4562 invite registration workflow refactor

Fixes # BLUI-4562 BLUI-4500 BLUI-4509 BLUI-4520



#### Changes proposed in this Pull Request:

- Fetch code and email from the url in invite registration workflow
- Added isInviteRegistration flag to RegistrationWorkflow component
- Added default screens for self registration and success registration screen
- Removed success screen passes as child from invite registration workflow example



#### Screenshots / Screen Recording (if applicable)

<img width="1494" alt="Screenshot 2023-08-02 at 4 17 14 PM" src="https://github.com/etn-ccis/blui-react-workflows/assets/120575281/61d2df58-0e9e-48fa-9dec-02201b0311d9">



#### To Test:

- **To test :** Edit login-workflowexample/Routing.tsx   ----- REGISTER_INVITE to have code and email as param
- e.g. **REGISTER_INVITE: '/register-by-invite?code=2&email=blui@eaton.com'**
- cd login-workflow
- yarn start
- Click on Debug button
- Click on Test Invite Register(http://localhost:3000/register-by-invite?code=2&email=blui@eaton.com)
- Go through the flow
- On success screen, you should see `email` passed in the url



#### Any specific feedback you are looking for?

- **Passing screens to registration workflow example with custom screens is for testing purpose and should be removed while merging**
